### PR TITLE
Added recipe for python-termstyle

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/python-termstyle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    # - setuptools
+
+test:
+  imports:
+    - termstyle
+
+about:
+  home: http://gfxmonk.net/dist/0install/python-termstyle.xml
+  license: BSD 3-Clause
+  # license_file: LICENSE not included in MANIFEST.in
```
Comment (jakirkham) [inline recipes/python-termstyle/meta.yaml]: Can we please add an issue or PR upstream to address this and then add it back to this comment?

File: recipes/python-termstyle/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    # - setuptools
```
Comment (jakirkham) [inline recipes/python-termstyle/meta.yaml]: Think we can drop this.

File: recipes/python-termstyle/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{%set name = "python-termstyle" %}
+{%set version = "0.1.10" %}
```
Comment (jakirkham) [inline recipes/python-termstyle/meta.yaml]: 0.1.11 😉

File: recipes/python-termstyle/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "f42a6bb16fbfc5e2c66d553e7ad46524ea833872f75ee5d827c15115fafc94e2" %}
```
Comment (jakirkham) [inline recipes/python-termstyle/meta.yaml]: This will obviously change then.

File: recipes/python-termstyle/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    - python
+
+test:
+  imports:
+    - termstyle
+
+about:
+  home: http://gfxmonk.net/dist/0install/python-termstyle.xml
+  license: BSD 3-Clause
+  # license_file: LICENSE not included in MANIFEST - see https://github.com/timbertson/termstyle/pull/9
```
Comment (jakirkham) [inline recipes/python-termstyle/meta.yaml]: Then we can drop the link to PR ( https://github.com/timbertson/termstyle/pull/9 ) and add the license. 😄

**Comment by pmlandwehr** [comment]
Crashed because of server capacity issues - closing and reopening.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**